src/common/turingHelper.js

```diff
@@ -1,7 +1,8 @@
 import _ from 'lodash';
 import { rpc, types, runtime } from '@oak-network/types';
-import { ApiPromise, WsProvider, Keyring } from '@polkadot/api';
+import { ApiPromise, WsProvider } from '@polkadot/api';
 import { u8aToHex } from '@polkadot/util';
+import Keyring from '@polkadot/keyring';
```

Comment:
The Keyring in @polkadot/api is a wrapper. Let’s use the real one underneath it instead.